apps/etterlatte-saksbehandling-ui/client/src/components/behandling/aktivitetsplikt/Aktivitetsplikt.tsx

```diff
@@ -0,0 +1,165 @@
+    })
+  }, [])
+
+  function edit() {
+    setAktivitetOppfolging(undefined)
+  }
+
+  return (
+    <Content>
+      {isFailure(hentet) && <ApiErrorAlert>En feil oppsto ved henting av data</ApiErrorAlert>}
+      {isFailure(lagret) && <ApiErrorAlert>En feil oppsto ved lagring av data</ApiErrorAlert>}
+
+      <ContentHeader>
+        <HeadingWrapper>
+          <Heading spacing size="large" level="1">
+            Oppfølging av aktivitet
+          </Heading>
+          <BodyShort spacing>
+            <strong>Dødsdato: </strong> {avdoedesDoedsdato}
+          </BodyShort>
+        </HeadingWrapper>
+      </ContentHeader>
+
+      <AktivitetspliktWrapper>
+        <Heading level="1" spacing size="medium">
+          Gjenlevende sin situasjon
+        </Heading>
+        <BodyLong spacing>
+          Ut ifra tid etter dødsfall stilles det ulike krav til aktivitet hos søker. Innen det har gått 6 måneder etter
+          dødsfall skal søker være i minst 50 % aktivitet for å motta omstillingstønad. Videre kan det forventes 100%
+          aktivitet etter 12 måneder.
+        </BodyLong>
+
+        <Heading size={'small'} spacing>
+          Beskriv etterlatte sin aktivitet idag
+        </Heading>
+
+        <Spinner visible={isPending(hentet)} label="Henter data" />
+
+        {!isPending(hentet) && aktivitetOppfolging && (
+          <SpacingWrapper>
+            <BodyLong>{aktivitetOppfolging.aktivitet}</BodyLong>
+            {!ferdigBehandlet && (
+              <Button variant="tertiary" icon={<PencilIcon />} onClick={() => edit()}>
+                Rediger
+              </Button>
+            )}
+            <Detail>Manuelt av {aktivitetOppfolging?.opprettetAv}</Detail>
+            <Detail>Sist endret {aktivitetOppfolging?.opprettet}</Detail>
+          </SpacingWrapper>
+        )}
+
+        {!isPending(hentet) && !aktivitetOppfolging && (
+          <>
+            <Textarea
+              label="Beskriv etterlatte sin aktivitet idag"
+              hideLabel={true}
+              size="medium"
+              value={beskrivelse}
+              onChange={(e) => setBeskrivelse(e.target.value)}
+            />
+            <SpacingWrapper>
+              <Button
+                variant="primary"
+                size="small"
+                className="button"
+                onClick={() =>
+                  lagre({ behandlingId: behandling.id, request: { aktivitet: beskrivelse } }, (lagretElement) =>
+                    setAktivitetOppfolging(lagretElement)
+                  )
+                }
+              >
+                Lagre beskrivelse
+              </Button>
+            </SpacingWrapper>
+          </>
+        )}
+
+        <Heading size={'small'} spacing>
+          Lag intern oppfølgingsoppgave med riktig frist
+        </Heading>
+        <BodyLong spacing>
+          Etterlatte skal følges opp og minnes på aktivitetskravet når det har gått 3-4 måneder og på nytt når det har
```

Comment:
Blir litt usikker på om disse oppgavene egentlig bør lages før saken er attestert, men det funker vel kanskje greit så lenge saksbehandler passer litt på. Hva tenker du?

Reply:
Tja, da må vel saken i prinsippet gå fra attestering til et nytt steg for saksbehandler, så sendes tilbake for iverksetting? Høres litt tungvint ut.. på sikt burde oppgaven(e) nok heller kanskje opprettes automatisk ved iverksetting.